### PR TITLE
upgrade Mvvmcross 5 for buildit mvvmcross

### PR DESCRIPTION
**Pull request by TheMiao**

### CODE REVIEW

File: src/BuildIt.Lifecycle/Samples/StateByState.UWP/project.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ﻿{
   "dependencies": {
-    "Autofac": "4.3.0",
+    "Autofac": "4.6.1",
+    "BuildIt.States": "1.1.0.48-beta",
```
Comment (builttoroamadmin) [inline src/BuildIt.Lifecycle/Samples/StateByState.UWP/project.json]: This shouldn't reference a beta version of states library

File: src/BuildIt.MvvmCross/BuildIt.MvvmCross.csproj [hunk 2]
```diff
@@ -0,0 +1,112 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (builttoroamadmin) [inline src/BuildIt.MvvmCross/BuildIt.MvvmCross.csproj]: This file is in the wrong location??? There is a sub folder where this belongs

Comment (TheMiao) [inline src/BuildIt.MvvmCross/BuildIt.MvvmCross.csproj]: Deleted file in wrong location

File: src/BuildIt.Lifecycle/Samples/StateByState.UWP/project.json (hunk 1 above)
Comment (TheMiao) [inline src/BuildIt.Lifecycle/Samples/StateByState.UWP/project.json]: It was originally in master branch sample app

**Pull request merged**